lib/config/rules/ecmascript-6.js

```diff
@@ -39,6 +39,8 @@ module.exports = {
   'prefer-arrow-callback': ['error', {allowNamedFunctions: true}],
   // Suggest using of const declaration for variables that are never modified after declared
   'prefer-const': 'warn',
+  // Require destructuring from arrays and/or objects <FEEDBACK>
+  'prefer-destructuring': ['error', {array: true, object: true}],
```

Comment:
One concern is that it'll force unstandardized syntax when a var is already defined.  e.g.,

```js
describe('foo', () => {
  let bar;

  beforeEach(() => {
    bar = sandbox.bar;
  });
```
... is fixed by rewriting line 5 to `({server} = sandbox);`.  Not great, but I think we declare+destructure more often than not.

(Regarding `array`, I've no idea what the final answer is, but I'm guessing that translating stuff like this would be a frustrating experience: `const totalSessions = results[0].result.data[0][0];`)